artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3934,6 +3943,14 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="max-size-messages" type="xsd:long" default="-1" maxOccurs="1" minOccurs="0">
```

Comment:
from the point of view of the config I prefer max-size-messages TBH

Reply:
It would be a fine name if it didnt as easily suggest totally different things hehe.

If not trying to contort to 'max-size-<foo>' for alignment to the other config, the most obvious thing for me would be just 'max-message-count'.